src/components/navbar/index.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from './navbar.module.scss';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+const Navbar = ({ page }) => {
+  const navBarContent = (title, refUrl) => {
+    const navBarText = title === 'Tasks' ? 'Index' : title;
```

Comment:
What is the purpose of 'Index'?

Reply:
When we click on the `Tasks` menu, it redirects us to the Index page .. So I am assuming that when we hit the url roadmap.realdevsquad.com/
the Index page will contain the Tasks details (The Index page)